### DISCUSSION TITLE
Start modifier for controllers suggestion

### DISCUSSION THREAD

**Discussion by fwannmacher** [diasurgical/DevilutionX#3966] [General]
![Screenshot from 2022-01-19 12-25-42](https://user-images.githubusercontent.com/1665660/150129208-7af9e9dc-4e56-43e3-81bd-bf525d7acf52.png)

now we have two circles with options for the start modifier but i question the need of some elements there. mainly MAP and MENU as we have already L3 and Start + Select for those. don't you think it would be better removing those and having only one circle?

**Comment by StephenCWills** [comment]
L3 isn't available on all controllers. For instance, that would make it impossible to open the map on 3DS.

As for Start+Select, I would argue that many people forget about that mapping because there is no in-game UI to guide them to that combination of buttons.

**Reply by fwannmacher** [reply]
good points
